### DISCUSSION TITLE
Game LoGo 🙂

### DISCUSSION THREAD

**Discussion by dinaolmelak** [Techwise-Team-15/my_pet#25] [Ideas]
![image-1](https://github.com/Techwise-Team-15/my_pet/assets/32272045/8668e9dc-36b4-4f21-bea5-cf2ad8bd9490)
---
*I asked AI to generate a picture for our game logo and it generated this 👆🤔. Feel free to comment below with your ideas for our game logo*
@Donnyd88 @kanan42 @RuthAnn1 @VanessaMedinaUNR

**Comment by RuthAnn1** [comment]
haha I like that picture.  It makes me smile.

**Reply by dinaolmelak** [reply]
😊 I like the art work and I had also gotten like an adventurous vibe from the pictures.

**Comment by dinaolmelak** [comment]
Just to Cement our decision made on the LoGo, Can you paste it here? @VanessaMedinaUNR

**Comment by VanessaMedinaUNR** [comment]
official game logo:
![logo1](https://github.com/Techwise-Team-15/my_pet/assets/132720091/54652edf-8082-480e-a802-8f4a6f9eea0b)

**Reply by VanessaMedinaUNR** [reply]
I also made it our team avatar @dinaolmelak

**Reply by dinaolmelak** [reply]
😱 Yes! Thank you, I love it!
